api/proto/teleport/devicetrust/v1/devicetrust_service.proto

```diff
@@ -439,8 +430,38 @@ message TPMEnrollChallenge {
 // calling the TPM2.0 `ActivateCredential` command on the client with the
 // parameters provided in `TPMEnrollChallenge`.
 message TPMEnrollChallengeResponse {
-  // The proof of possession of both the EK and AK.
+  // The client's solution to `TPMEncryptedCredential` included in
+  // `TPMEnrollChallenge` using ActivateCredential.
   bytes solution = 1;
+  // The result of the client's platform attestation with the nonce provided
+  // in `TPMEnrollChallenge`.
+  TPMPlatformParameters platform_parameters = 2;
+}
+
+// Encapsulates the value of a PCR in a point at time.
+// See https://pkg.go.dev/github.com/google/go-attestation/attest#PCR
+message TPMPCR {
+  int64 index = 1;
+  bytes digest = 2;
+  uint64 hash = 3;
```

Comment:
Comment that "hash" matches the crypto.Hash enum?

Reply:
Good catch, forgot to copy the comment over for this block.